scripts/load-pretty-printers.in

```diff
@@ -0,0 +1,2 @@
+source @Thrust_SOURCE_DIR@/scripts/gdb-pretty-printers.py
```

Comment:
What is a .in file?

Reply:
Not sure if this is a general convention, but in the projects I worked in, we use it for template files that will be expanded into an actual file by CMake - config headers, pkg-config files etc.

Reply:
The CMake convention is `<file>.suffix.in` for the input to `configure_file`. ( `.cpp.in` => `.cpp` )